examples/ic_api/src/ic_api.ts

```diff
@@ -11,6 +12,16 @@ import {
     Update
 } from 'azle';
 
+// returns the argument data as bytes.
+export function arg_data_raw(
```

Comment:
Let's add one record and one enum as well

Reply:
There are issues with records and variants. I've made https://github.com/demergent-labs/azle/issues/487 to track it.